configure.php

```diff
@@ -1001,13 +1001,21 @@ function xinclude_residual( DOMDocument $dom )
     // XInclude failures are soft errors on translations, so remove
     // residual XInclude tags on translations to keep them building.
 
+    $header = false;
     $explain = false;
 
     $xpath = new DOMXPath( $dom );
     $xpath->registerNamespace( "xi" , "http://www.w3.org/2001/XInclude" );
     $nodes = $xpath->query( "//xi:include" );
     foreach( $nodes as $node )
     {
+        if ( $header == false )
```

Comment:
What kind of formatting code style are you following? This line isn't following the same style as the line before it.